content/modules/_index.md

```diff
@@ -1,22 +1,15 @@
 ---
```

Comment:
I decided that there was too much clutter on the landing page. What do you think?
https://docs.redislabs.com/staging/RED-32998-modules-section/modules/
(This is the last change... really)

Reply:
@K-Jo How about:
Redis Labs develops, certifies, and packages modules for [redis](redis.io), [Redis Enterprise Software (RS)]({{< relref "/rs" >}}) clusters and [Redis Cloud Pro]({{< relref "/rv" >}}). These certified modules are bundled with RS and RC: